### ISSUE TITLE
Images Not Uploading

### ISSUE THREAD

**Issue by jessicagood** [StraboSpot/strabo-mobile#272]
From @egbunse After going through all the field camp datasets to test the plug-in I've found four accounts which still will not upload the images to the online server for use. I can access these images from connecting to the offline server and using my REST client, and can see them on the app, so they should be getting uploaded.
Both the app and iPads have been updated as much as possible (0.13.4 and the latest Apple update), but on these four accounts the 'problem datasets' never say they've been successfully uploaded in the app. I'm no longer getting the error message for images we got last week. 
Here is a list of the accounts and problem datasets: 
fc7: temple canyon 
fc16: blue ridge and temple canyon 
fc17: blue ridge and temple canyon 
fc20: temple canyon 
Doug is working on uploading these to iMazing this morning. Please let me know if you need more info.

**Comment by jessica-azgs** [comment]
@egbunse I uploaded blue ridge from fc17 and downloaded the dataset on another device and the images seemed to come in fine. Were there specific images that were a problem or maybe I should try a different project/dataset?

**Comment by egbunse** [comment]
@jessica-azgs I tried getting images for fc17 on my rest client after you sent that, and it now works perfectly. So I went back and tried to upload the others again from the iPads, and so far all except fc7 have been successfully uploaded and I can see the images (no idea why they couldn't properly upload all yesterday afternoon!) If you could try uploading the fc7 on your end, I'll stop trying to over here (it has been going for about 10 mins without a success message).

**Comment by jessica-azgs** [comment]
fc7 uploaded ok for me. Though there is one spot that is not downloading. It might be related but I can't open the excel files at strabospot.org for either blue ridge or temple canyon in fc7. blue ridge uploads and downloads fine. The one spot that is not downloading is 14664357196552

**Comment by jessica-azgs** [comment]
Actually maybe it is downloading but there is something wrong with it. Still looking at it.

**Comment by jessica-azgs** [comment]
@egbunse Can you try the latest version 0.13.6? This was related to the problem we had earlier in the week when the images didn't upload/download correctly. You may need to switch out of the project and then back in to make sure all of the images download.

@jasonash I'm going to close this once @egbunse is able to test that it works. (Or actually you can just close it @egbunse, if it works.) The Excel files still aren't opening however, but seems like that is a different issue.

**Comment by egbunse** [comment]
@jessicagood After updating the iPad app to 0.13.6, the Temple Canyon dataset successfully uploaded to the server. Then I was able to use my rest client to see images from that dataset. I successfully downloaded that dataset onto my phone, but the images don't actually show up there- perhaps a lack of storage issue on my phone though. So I used both the online app and a spare iPad and could successfully download the dataset and view images on those. I'm thinking this issue is resolved, thanks for your help!
